configs/katello/3.15.yaml

```diff
@@ -0,0 +1,57 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+:releases:
+  :3.15.0:
+    :redmine_version_id: 1131
+:code_name: Rocky Road
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.15
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.0
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 0.21.1
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.3
+    :repo: https://github.com/Katello/katello-host-tools.git
+:ignores:
+:strict_keys: true
+:gpg_key: 06ABEFE2
+:mash_scripts:
+:tags:
+  - name: katello-3.15-thirdparty-candlepin-rhel7
+    based_off: katello-thirdparty-candlepin-rhel7
+    arches:
+      - x86_64
+  - name: katello-3.15-rhel7
+    based_off: katello-nightly-rhel7
+    helper_tags:
+      katello-3.15-thirdparty-rhel7: katello-thirdparty-rhel7
+      katello-3.15-rhel7-override: null
+    build_target: katello-3.15-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
+      katello-3.15-rhel7-build:
+        katello-3.15-rhel7-override: 0
+        foreman-rails-2.0-rhel7: 2
```

Comment:
This no longer exists

Reply:
@ekohl did you mean the whole block, or just the "formeman-rails"?

Reply:
Just this line. The rails repository was merged into Foreman itself. This probably needs to be done to the nightly tag as well.

Reply:
In light of this change it looks like the procedures need to be updated: https://github.com/theforeman/tool_belt/blob/674abe26cba076a94eada9f713799091edd1824c/procedures/katello/branch.md.erb#L13-L14